index.html

```diff
@@ -70,9 +70,15 @@
             <img class="logo" src="img/github.png" alt="Github Icon" />
           </button>
         </a>
+
         <div class="user-greet">
-          <div class="user-greet-msg">Hello text</div>
-          <img class="user-profile-pic" />
+          <div class="user-greet-hello">Hello,</div>
+          <a href="https://my.realdevsquad.com/">
+            <div class="user-greet-msg">username</div>
+          </a>
+          <a href="https://my.realdevsquad.com/">
+            <img class="user-profile-pic" />
+          </a>
```

Comment:
Can we combine this to just one reference?

Reply:
Sure. On it